tools/run_tests/artifacts/distribtest_targets.py

```diff
@@ -108,7 +108,7 @@ def __init__(self,
     def pre_build_jobspecs(self):
         return []
 
-    def build_jobspec(self):
+    def build_jobspec(self, inner_jobs=None):
```

Comment:
Python style nit: For unused argument, we can add prefix, like `unused_inner_jobs`, or add `del inner_jobs` at the first line. https://google.github.io/styleguide/pyguide.html#214-decision